ui/src/app/settings/containers/Settings/Settings.js

```diff
@@ -1,10 +1,15 @@
 import React from "react";
+import { useDispatch } from "react-redux";
 
+import actions from "app/settings/actions";
 import Routes from "../../components/Routes";
 import Section from "app/base/components/Section";
 import SettingsNav from "../../components/Nav";
 
 const Settings = () => {
+  const dispatch = useDispatch();
+  dispatch(actions.config.fetch());
```

Comment:
Same comment as for Steve, but you'll need to read up about lifecycle management with hooks and [`useEffect`](https://reactjs.org/docs/hooks-effect.html). Also have a look at the [`useFetchOnce`](https://github.com/canonical-web-and-design/maas-ui/blob/2d85ce8fd9ce0441c9f1a23c1679d0fe8a96fad3/ui/src/app/settings/containers/Users/Users.js#L56) hook, which will be replaced soon, but is okay to use for now.